Cargo.toml

```diff
@@ -15,7 +15,7 @@ exclude = [
 
 [dependencies]
 # Private dependencies.
-libc = "^0.2.99"
```

Comment:
We'd need `rustix` to carry this lower bound for us, in order to avoid regressing https://github.com/lucab/memfd-rs/pull/25.

Reply:
Yes, it does this. rustix depends on at least libc 0.2.114 because that's the version that has my patch adding `MFD_*` constants to libc, which the features used in this PR depend on :-).